.travis.yml

```diff
@@ -101,11 +101,12 @@ script:
   - export GO_TEST_TIMEOUT=20m
   - |
     if [[ "${PRESUB_TESTS}" == "true" ]]; then
-      ./scripts/presubmit.sh ${WITH_COVERAGE:+--coverage}
+      ./scripts/presubmit.sh ${PRESUBMIT_OPTS}
       # Check re-generation didn't change anything. Skip protoc-generated files
       # because protoc is not deterministic when generating file descriptors.
+      # Skip go.mod and go.sum because testing may add indirect dependencies that would be trimmed by 'go mod tidy'
       echo "Checking that generated files are the same as checked-in versions."
-      git diff --exit-code -- ':!*.pb.go' ':!*_string.go'
+      git diff --exit-code -- ':!*.pb.go' ':!*_string.go' '!go.*'
```

Comment:
I couldn't quickly find a spec for this path expression. Just to double check, should there not be a colon before the exclamation mark?

Reply:
Sharp eye

I don't see any reference to a colon here:
https://git-scm.com/docs/gitignore#_pattern_format

Reply:
That's because this is a [pathspec](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefpathspecapathspec), not a gitignore pattern.